### DISCUSSION TITLE
Add CI section when installing git-revision-date-localized

### DISCUSSION THREAD

**Discussion by mendax1234** [squidfunk/mkdocs-material#6013] [Ideas]
Hi, l find that if you want to show the created date of a post using git-revision-date-localized plugin, you should also configure the CI section. Only adding two lines in `mkdocs.yml` is not enough. [See this link](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/issues/118#issuecomment-1719284728)

Hope that the related part can be added to the document! Besides, l am not sure whether this project supports pull request from outsiders who are not collaborators. Since when l try use venv to test the mkdocs documentation locally, l find that after running `pip install -r requirements.txt`, l still get something wrong after running `mkdocs serve`. And the wrong message is `ERROR   -  Config value 'plugins': The "minify" plugin is not installed`

BTW, hope that the part can be added !

**Comment by alexvoss** [comment]
Hi, I can answer the question about accepting pull requests from people who are not collaborators. I have done my first one a few days ago - likewise was a small fix/addition to the documentation. It seemed like a bit much effort but then I think it is good to see how many people are chipping in to a project, so I would encourage you to go for it.  (I may show up as a collaborator now but wasn't at the time.)

**Reply by mendax1234** [reply]
Thanks for your reply! l will try to make a PR! But l run into some problems when testing my changes on the local device, and I am not a collaborator, so l am wondering whether the existing GitHub folder may not contain the whole stuff l need to build and test locally.

**Comment by squidfunk** [comment]
Our [environment setup guide](https://squidfunk.github.io/mkdocs-material/customization/#environment-setup) lists all dependencies that need to be installed:

```
pip install -e .
pip install mkdocs-minify-plugin
pip install mkdocs-redirects
pip install nodeenv
```

We'll be looking into how we can make this simpler in the near future as we revisit the customization guide. That being said, we can add a note on the CI section to our documentation, so if you want to create a PR, you're invited to do so.

**Reply by mendax1234** [reply]
Got it! Thank you!

But l think l will just work on the CI part of `git-revision-date-localized`. 😂
